packages/devkit/src/utils/config-utils.ts

```diff
@@ -71,7 +72,15 @@ async function load(path: string): Promise<any> {
   } catch (e: any) {
     if (e.code === 'ERR_REQUIRE_ESM') {
       // If `require` fails to load ESM, try dynamic `import()`.
-      return await dynamicImport(`${path}?t=${Date.now()}`);
+      try {
+        return await dynamicImport(`${path}?t=${Date.now()}`);
+      } catch (e) {
+        if (e.code === 'ERR_UNSUPPORTED_ESM_URL_SCHEME') {
```

Comment:
Why do we have to `try catch` this? Can't we ensure that the URL scheme is supported?

Reply:
I will do some investigations

Reply:
Yes we can.